src/inputs/Hidden.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import autoBindMethods from 'class-autobind-decorator';
+import { omit } from 'lodash';
+
+import * as Antd from 'antd';
+
+import {
+  IAntFormField,
+  IInjected,
+  IInputProps,
+} from '../interfaces';
+
+@autoBindMethods
+@observer
+class Hidden extends Component<IInputProps> {
```

Comment:
`HiddenInput`? Just `Hidden` sounds too generic, in fact, after reading the name I expected a component to hide any children based on a condition or something. But then noticed that his is actually an input.

Reply:
We do Rate and Date without input 🤔

Reply:
I think we should rename them too... 😊 What are they? Date Picker? Date range? Date only? Date and time? Rate gauge? Rate like in 5 stars? Is it an input or a control? But for the scope of this particular PR Hidden is just too ambiguous.

Reply:
All of these are inputs. They are in the `inputs/` folder, but if you think we should siffix all of them with `*Input.tsx` I could understand that:

![image](https://user-images.githubusercontent.com/796717/58482127-d2b27b80-812b-11e9-954e-9d7fdc792dca.png)

Reply:
That's fine, I didn't notice they where in the inputs folder. That should be clear enough.